src/twisted/trial/_dist/disttrial.py

```diff
@@ -165,7 +166,7 @@ class WorkerPool:
     def _createLocalWorkers(
         self,
         protocols: Iterable[LocalWorkerAMP],
-        workingDirectory: FilePath,
+        workingDirectory: FilePath[Any],
```

Comment:
```suggestion
        workingDirectory: FilePath[str],
```

called `workingDirectory.child(str(x))`

Reply:
ah no this should still be `FilePath[Any]` child is defined as`def child(self, name: OtherAnyStr) -> FilePath[OtherAnyStr]: ...`